packages/types/src/inventory/common/inventory-level.ts

```diff
@@ -53,7 +54,8 @@ export interface InventoryLevelDTO {
   deleted_at: string | Date | null
 }
 
-export interface FilterableInventoryLevelProps {
+export interface FilterableInventoryLevelProps
+  extends BaseFilterable<FilterableInventoryLevelProps> {
```

Comment:
Q: Should we allow OperatorMap on the props? Also, why do we use NumericalComparisonOperator which is using another type of operator and why don't we rely on OperatorMap instead for those specific operator. It seems weird to have two differet type of possible operators for our modules e.g lt vs $lt